tools/sgen/input/FluentUIStyle.yml

```diff
@@ -239,6 +239,14 @@ MSFGhostButtonTokens extends MSFButtonTokens:
   borderColor: [ rest: $Colors.Neutral.clear, hover: $Colors.Neutral.clear, pressed: $Colors.Neutral.clear, selected: $Colors.Brand.shade10, disabled: $Colors.Neutral.clear ]
 
 AP_MSFListTokens:
+  backgroundColor: [ rest: $Colors.Background.neutral1, pressed: $Colors.Background.neutral5 ]
+  borderColor: $Colors.Stroke.neutral1
+  borderSize: $Border.size.thin
+  horizontalCellPadding: $Spacing.medium
+  sublabelColor: $Colors.Foreground.neutral3
```

Comment:
What's the goal of having the sublabel font and color on the list since they're already on the cell tokens?
And why only the sublabels and not the labels?

Reply:
The sub label tokens are used by header/footer, which will get their own AP from the next PR, but adding that right now would cause overlap... this will get fixed in #424 . For now I am letting the headers get their tokens from the List AP